examples/rust/get_started/examples/06-credentials-exchange-client.rs

```diff
@@ -0,0 +1,77 @@
+use ockam::authenticated_storage::AuthenticatedAttributeStorage;
+use ockam::identity::credential_issuer::{CredentialIssuerApi, CredentialIssuerClient};
+use ockam::identity::{Identity, SecureChannelTrustOptions, TrustEveryonePolicy};
+use ockam::sessions::Sessions;
+use ockam::{route, vault::Vault, Context, Result, TcpConnectionTrustOptions, TcpTransport};
+use std::sync::Arc;
+
+#[ockam::node]
+async fn main(mut ctx: Context) -> Result<()> {
+    // Initialize the TCP Transport
+    let tcp = TcpTransport::create(&ctx).await?;
+
+    // Create an Identity representing the client
+    // We preload the client vault with a change history and secret key corresponding to the identity identifier
+    // Pe92f183eb4c324804ef4d62962dea94cf095a265d4d28500c34e1a4e0d5ef638
+    // which is an identifier known to the credential issuer, with some preset attributes
+    let vault = Vault::create();
+
+    // Create an Identity representing the server
+    // Load an identity corresponding to the following public identifier
+    // Pada09e0f96e56580f6a0cb54f55ecbde6c973db6732e30dfb39b178760aed041
+    //
+    // We're hard coding this specific identity because its public identifier is known
+    // to the credential issuer as a member of the production cluster.
+    let identity_history = "01dcf392551f796ef1bcb368177e53f9a5875a962f67279259207d24a01e690721000547c93239ba3d818ec26c9cdadd2a35cbdf1fa3b6d1a731e06164b1079fb7b8084f434b414d5f524b03012000000020a0d205f09cab9a9467591fcee560429aab1215d8136e5c985a6b7dc729e6f08203010140b098463a727454c0e5292390d8f4cbd4dd0cae5db95606832f3d0a138936487e1da1489c40d8a0995fce71cc1948c6bcfd67186467cdd78eab7e95c080141505";
+    let secret = "41b6873b20d95567bf958e6bab2808e9157720040882630b1bb37a72f4015cd2";
+    let client = Identity::create_identity_with_history(&ctx, vault, identity_history, secret).await?;
```

Comment:
let's rename the function `create_identity_with_history` to `create_identity_with_change_history`